docker_base/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# This is a base container we use. It has pgsql, redis and Python 3.5,
+# plus a user named qabel and locales.
+
+FROM ubuntu:16.04
+MAINTAINER Marian Beermann <beermann@qabel.de>
+RUN apt-get update && \
+    apt-get install -y postgresql-9.5 libpq-dev \
+                       redis-server redis-tools \
+                       build-essential libffi-dev git \
+                       python3.5 python3-virtualenv virtualenv python3-pip && \
```

Comment:
Best practice (according to the docs) is to sort the arguments alphanumerically and to put every package in a new line. I understand the blocks for python, postgres and redis but you could split the build tools (because they are probably the first to get extended)